wee_slack.py

```diff
@@ -2925,6 +2925,26 @@ def msg_command_cb(data, current_buffer, args):
             channel.send_message(message)
     return w.WEECHAT_RC_OK_EAT
 
+@slack_buffer_required
+def command_channels(data, current_buffer, args):
```

Comment:
Please decode data and args from utf8 byte strings to unicode strings like it is done in the other functions. This goes for `command_users` as well. I.e. like this:

```
     data = decode_from_utf8(data)
     args = decode_from_utf8(args)
```

Reply:
@trygveaa  i need to decode data even when i'm not using this parameter?

Reply:
The reason I asked for this is that it's easy to forget if the functions were changed to use the arguments later. But it's inconvenient and ugly, so I created a PR with a decorator for it instead. So you don't need to do anything in regards to this.